ui/coordinateTrainer/src/chessground.ts

```diff
@@ -10,6 +10,9 @@ export default function (ctrl: CoordinateTrainerCtrl): VNode {
       insert: vnode => {
         const el = vnode.elm as HTMLElement;
         ctrl.chessground = site.makeChessground(el, makeConfig(ctrl));
+        site.pubsub.on('theme.change', () => {
+          ctrl.chessground!.state.addPieceZIndex = $('#main-wrap').hasClass('is3d');
```

Comment:
instead of reading state from DOM outside the module, get a `is3d` boolean argument from the event itself